egs/apiai_decode/s5/download-model.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Downlaods Api.ai chain model into exp/api.ai-model/ (will replace one if exists)
+
+DOWNLOAD_URL="https://api.ai/downloads/api.ai-kaldi-asr-model.zip"
+
```

Comment:
I'm a little concerned that this URL you have chosen gives you no capability for increasing version numbers (e.g. after retraining) or adding more languages.  Don't you want to structure it in such a way that it's trivial to extend to newly created models, and other languages?